serverlessworkflow/modules/ROOT/pages/cloud/operator/customize-podspec.adoc

```diff
@@ -194,15 +194,20 @@ For more information about configuring workflows see xref:cloud/operator/configu
 
 When setting the attribute `.spec.podTemplate.container.image` the operator understands that the workflow already have an image built and the user is responsible for the build and image maintainence. That means that the operator won't try to upgrade this image in the future or do any reconciliation changes to it.
 
+[IMPORTANT]
+====
+The image is expected to contain the workflow definition it is based on. This workflow should be located in /deployments/app/ and have the format *.sw.{json|yaml}. It is also important to remember that the spec.flow defined in the workflow must match the one in the image.
+====
+
 === Setting a custom image in devmode
 
-In xref:cloud/operator/developing-workflows.adoc[development profile], it's expected that the image is based on the default `quay.io/kiegroup/kogito-swf-devmode:latest`. 
+In xref:cloud/operator/developing-workflows.adoc[development profile], it's expected that the image is based on the default `quay.io/kiegroup/kogito-swf-devmode:latest`.
```

Comment:
@ricardozanini shouldn't this be the `https://hub.docker.com/r/apache/incubator-kie-sonataflow-devmode/tags` url?

Reply:
Yes!